src/components/SearchInput/SearchInput.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useEffect, useContext, useRef } from 'react'
+import PropTypes from 'prop-types'
+import { Button, FormGroup, Label, Input, InputGroup, InputGroupText } from 'reactstrap'
```

Comment:
Instead of using reactstrap and needing to duplicate styles and css variables in the new files below - we should just import these components from the cms.gov design system which is already installed as a peer dependency on this repo.

@cmsgov/design-system